src/NSubstitute.Analyzers.Shared/Extensions/SyntaxGeneratorExtensions.cs

```diff
@@ -21,5 +21,15 @@ internal static class SyntaxGeneratorExtension
 
             return invocationExpression;
         }
+
+        public static SyntaxNode InternalVisibleToDynamicProxyAttributeList(this SyntaxGenerator syntaxGenerator)
+        {
+            var attributeArguments =
+                syntaxGenerator.AttributeArgument(syntaxGenerator.LiteralExpression("DynamicProxyGenAssembly2"));
+
+            return syntaxGenerator.Attribute(
+                "System.Runtime.CompilerServices.InternalsVisibleTo",
```

Comment:
I haven't tested this, but can `nameof` be used here?

Reply:
This will not give you qualified name, only `InternalsVisibleToAttribute`. I was thinking about changing this to typeof(InternalsVisibleToAttribute).FullName but this will result in generating `InternalsVisibleToAttribute` instead of shorter one `InternalsVisibleTo`.